webpack.config.dev.js

```diff
@@ -12,15 +12,12 @@ const insertHotReload = (host, port, entries) => (
   R.map(R.compose(R.flatten, v => [v].concat(hotEntry(host, port))), entries)
 );
 
-const devConfig = {
+const devConfig = Object.assign(config, {
```

Comment:
I need some way to add the development specific config though, what exactly are you suggesting?